certified-connectors/Zellis/README.md

```diff
@@ -0,0 +1,18 @@
+# Title
+Zellis HCM contains operations to allow you to easily send and receive employee information between Zellis HCM and your other systems. 
+
```

Comment:
Please modify readme.md file as per the template in the below link. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
Have made the changes and update the PR

Reply:
The changes are not reflected.  Please check.